subjects/devops/orchestrator/README.md

```diff
@@ -0,0 +1,145 @@
+
+You must install `kubectl` on your machine to manage your cluster.
+
+The nodes must be connected and available!
+
+```console
+$> kubectl get nodes -A
+NAME                                           STATUS   ROLES    AGE    VERSION
+<master-node>   Ready    <none>   XdXh   vX
+<agent1-node>   Ready    <none>   XdXh   vX
+$>
+```
+
+You must provide a `orchestrator.sh` script that run and create and manage the infrastructure
+
+```console
+$> ./orchestrator create
+cluster created
+$> ./orchestrator start
+cluster started
+$> ./orchestrator stop
+cluster stoped
+$>
+```
+
+### Dockerhub
+
+You will need to push the Docker images for each component to Docker Hub.
+
+> You will use it in your kubernetes manifests.
+
+![Dockerhub-example](pictures/dockerhub-example.jpg)
+
+### Manifests
+
+You should create a YAML Manifest that describes each component or resource of your deployment.
+
+### Secrets
+
+You must store your passwords and credentials as a k8s secrets
+
+> It's forbidden to put your passwords and credentials, in the YAML manifests, except the secret manifests!
+
+### Node.JS apps
+
+Your Node.JS applications must be deployed as a deployment and they must be scaled horizontally automatically, depending on CPU consumption.
+
+- `api-gateway`:
+  max replication: 3
+  min replication: 1
+  cpu percent triger: 60%
+
+- `billing-app`:
+  max replication: 3
+  min replication: 1
+  cpu percent triger: 60%
+
+- `inventory-app`:
+  max replication: 3
+  min replication: 1
+  cpu percent triger: 60%
+
+### Databases
+
+Your databases must be deployed as Statefulset in your k3s cluster, and you must create volumes that enable containers to move across infrastructure without losing the data.
+
+### Documentation
+
+You must push a `README.md` file containing full documentation of your solution (Prerequisites, Configuration, Setup, Usage, ...).
+
+### Bonus
+
+If you complete the mandatory part successfully and you still have free time, you can implement anything that you feel deserves to be a bonus, Examples:
+
+- Deploy a Kubernetes Dashboard to monitor the cluster
+
+- Deploy a dashboard for applications logs
+
+- Kubernetes in cloud ?!
+
+Challenge yourself!
+
+### Submission and audit
+
+You must submit the `README.md` file and all files used to create and delete and manage your infrastructure: Vagrantfile, Dockerfiles, Manifests,...
+
+```console
+.
+├── Manifests
+│   └── [...]
+├── Scripts
+│   └── [...]
+├── Dockerfiles
+│   └── [...]
+└── Vagrantfile
+```
+
+If you decide to use a different structure for your project remember you should be able to explain and justify your decision during the audit.
+
+> In the audit you will be asked different questions about the concepts and the practice of this project, prepare yourself!
+
+#### What next?
+
+In order to develop your knowledge and career as a devops engineer, we highly recommend you to learn and practice more about of kubernetes and even get a certification for kubernetes.

```

Comment:
`... more about kubernetes` (removing the `of`).

Reply:
fixed